decimal-counter/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    var myString = "190.9876"
+    decimalPlaceCounter(myString)
+    function decimalPlaceCounter(stringNumber){
+     for(var i = 0; i<stringNumber.length; i++){
+       if(stringNumber[i]=="."){
+         var decPlace = i
+         console.log("The string:", myString, " has ", stringNumber.length-decPlace-1, " numbers after the decimal.")
+       }
+     }
+    }
```

Comment:
This works but you want to return the result out of the function, not just console log it.